File: 프로그래머스/lv1/12901. 2016년/2016년.java
class Solution {
    public String solution(int a, int b) {
        String[] day = new String[]{"THU","FRI","SAT","SUN","MON","TUE","WED"}; 
        int[] daysum = new int[]{31, 29, 31,30, 31, 30, 31,31, 30, 31, 30, 31};
        int cal=0;
         { for( int i =0; i <a-1;i++){cal += daysum[i];}}
         return day[(cal+b) % 7];
        
    }
}

File: 프로그래머스/1/12901. 2016년/2016년.py
def solution(a, b):
    month = [i for i in range(1,13)]
    day = [31,29,31,30,31,30,31,31,30,31,30,31]
    week = ['FRI','SAT','SUN','MON','TUE','WED','THU']
    return week[(sum(day[:a-1])+b-1)%7]